hindi/swami-vivekananda/swami-vivekananda.md

```diff
@@ -1 +1,2 @@
 ﻿पता है स्वामी विवेकानंद जी क्या कहते थे? [यहाँ देखो|](teachings-of-swami-vivekananda/quotes.md)
+स्वामी विवेकानंद जी की संक्षिप्त जीवनी [यहाँ है|](Introduction-of-Swami-vivekananda/introduction.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.